lib/webdrivers/chromedriver.rb

```diff
@@ -30,28 +30,30 @@ def latest_version
 
       private
 
+      def platform
+        if Selenium::WebDriver::Platform.linux?
```

Comment:
How about we return `Selenium::WebDriver::Platform` here and then call `#linux?`, `#mac?`, and `#linux?` on it instead of comparing strings in `#file_name` and `#chrome_version`?

Reply:
This will also fix the rubocop offense on line 35.

Reply:
we can just call `#windows?`, and it's definitely the more precise implementation for this. Updated.